consensus/fork_choice/src/fork_choice.rs

```diff
@@ -1192,7 +1192,7 @@ where
         // Update store time.
         store.set_current_slot(time);
 
-        let current_slot = store.get_current_slot();
+        let current_slot = store.get_current_slot();    
```

Comment:
There's some trailing whitespace here

```suggestion
        let current_slot = store.get_current_slot();
```

Reply:
Done, `cargo fmt` didn't report anything for me ..